scripts/docker-develop/Dockerfile

```diff
@@ -1,7 +1,10 @@
-FROM adoptopenjdk:8
+FROM ubuntu:focal
```

Comment:
Why use `ubuntu` and then manually install adoptopenjdk instead of using adoptopenjdk image directly?

Reply:
I think @aednichols found that starting with ubuntu and installing openjdk was faster than starting with the openjdk image.